cli/app.py

```diff
@@ -63,9 +51,12 @@ def create(cls, app_name, config, args=None):
         """
         settings_dir = config.get('defaults', 'settings')
 
-        if app_name in ["provisioner", ]:
+        if app_name not in ["provisioner", "installer"]:
```

Comment:
instead of hard-code the "provisioner/installer/etc" support, I would add it to the conf (i.e: supported_specs: <list>) and this way the user defines dynamically what objects get created based on the configuration.